### DISCUSSION TITLE
Why am I - spontaniously - getting a notification everytime I'm saving my buffer?

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#4628] [General]
I'm getting this notification asking me if I want to overwrite the existing file:

<img width="754" alt="Screenshot 2024-10-22 at 23 03 08" src="https://github.com/user-attachments/assets/9787af0a-731d-4bac-8747-f85d5c386ea7">

Why is that? and why am I getting this _sometimes_?

**Comment by idelice** [comment]
Important note:

It seems that it _only_ happens when I enable `blink.cmp` and disable `nvim-cmp`:

```lua
  { "hrsh7th/nvim-cmp", enabled = false },
  { "saghen/blink.cmp", enabled = true },
```

**Comment by gazofnaz** [comment]
I had the same issue - every (markdown) file I tried to change resulted in the "overwrite existing file" pop-up appearing.

In my case, it was caused by this change in my `autocmds.lau` file:

```
-- disable completion on markdown files by default
vim.api.nvim_create_autocmd("FileType", {
  pattern = { "gitcommit", "markdown" },
  callback = function()
    require("cmp").setup({ enabled = false })
  end,
})
```

TBH I don't know how to fix it properly so I've just removed that change and left auto-complete enabled in my markdown files. It's a bit annoying, but I've not got the time to investigate further at this stage. I'll respond with a full fix if I find one.
